returnn/tf/layers/basic.py

```diff
@@ -7259,10 +7259,11 @@ def __init__(self, factor, axis, out_dim=None, kind="nn", fill_value=None, fill_
     out_dim  # noqa  # via get_out_data_from_opts
     super(ResizeLayer, self).__init__(**kwargs)
     # self.output.shape and self.output.batch_dim_axis are already set here via self.get_out_data_from_opts().
-    axis = self.output.get_axis_from_description(axis)
-    assert axis > 0, "batch-dim resize not supported"
-    assert self.input_data.batch_ndim == 3, "not implemented otherwise"
-    input_data = self.input_data.copy_as_batch_major()
+    axis_ = self.input_data.get_axis_from_description(axis)
```

Comment:
Wrong order. First you should do `input_data = self.input_data.copy_as_batch_major()`, then you can call `get_axis_from_description`.